### ISSUE TITLE
Shouldn't warn when using modules

### ISSUE THREAD

**Issue by SerjoPepper** [davidjbradshaw/iframe-resizer#462]
Hello, I use iframe-resizer as a module (not a global window object). So when my middle iframe (both contentWindow and source window) receive message, I see next warn:

```
iFrameSizer][iFrameResizer0] Unexpected message ([iFrameSizer]iFrameResizer0:72:6841:init)

```

So I think next code should check usage of node_modules/es6_modules:
```
			function callFromParent(){
				var messageType = getMessageType();

				if (messageType in processRequestFromParent){
					processRequestFromParent[messageType]();
				} else if (!isMiddleTier() && !isInitMsg()){
					warn('Unexpected message ('+event.data+')');
				}
			}
```

should be

```
			function callFromParent(){
				var messageType = getMessageType();

				if (messageType in processRequestFromParent){
					processRequestFromParent[messageType]();
				} else if (!isMiddleTier() && !isInitMsg() && typeof module === 'undefined' && typeof module.exports === 'undefined')){
					warn('Unexpected message ('+event.data+')');
				}
			}
```

**Comment by davidjbradshaw** [comment]
Interesting question. I'm half wondering if we shouldn't just get rid of the warning completely.

It's not possible for the `typeof` `module` and `module.exports` to both be `undefined`.

Feel free to have a play around and submit a PR if you get if working.

**Comment by davidjbradshaw** [comment]
Released in v3.5.9
